Sources/KituraSession/CookieCryptography.swift

```diff
@@ -40,12 +40,12 @@ class CookieCryptography {
     ///
     private let originalLength = 36
 
-    init (secret: String) {
+    init (secret: String) throws {
```

Comment:
Isn't this a breaking change? All users of this API will have to `do {} catch {}` it now?

I would rather not have to declare a new major version of this package, so I suggest that instead of marking this function as `throws` we wrap the `PBKDF.deriveKey()` calls in `do {}` instead, and if they throw then we just `fatalError()` (which is what Cryptor used to do anyway...)

Reply:
It is an internal initialize that is not part of the public API, so it shouldn't affect anyone other than ourselves. We avoid surfacing this exception in the constructor for Session by catching and logging it and then calling `fatalError()`, which is what BlueCryptor previously did internally.

Reply:
Great, thanks for explaining.  I missed that it's actually `internal` even though the type is `public`.